ledgers-bank-account-access-management/ledgers-bank-account-access-repository/src/test/java/de/adorsys/ledgers/baam/db/domain/SeniorManagerAccessTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2018-2023 adorsys GmbH and Co. KG
+ * All rights are reserved.
+ */
+
+package de.adorsys.ledgers.baam.db.domain;
+
+import org.junit.jupiter.api.Test;
+class SeniorManagerAccessTest {
+
+    @Test
+    void createManagerRole() {
+    }
+
+    @Test
+    void modifyManagerRole() {
+    }
+
+    @Test
+    void revokeManagerRole() {
+    }
+
+    @Test
+    void suspendAccess() {
+    }
+
+    @Test
+    void activateAccess() {
+    }
+}
```

Comment:
Hello @Koufan-De-King please implememt the various test cases ..

Also since the seniormanageraccess is an entity, it needs to be included into the ledgers DB..

to verify, executed ```docker compose up``` on the ledgers root directory to see..